qmxgraph/widget.py

```diff
@@ -454,6 +464,11 @@ def _on_drop(self, event):
                         tags=v.get('tags', {}),
                     )
 
+            if version in (3,):
+                custom_app_drop_events = parsed.pop('custom_app_drop_events', [])
```

Comment:
Could you mention `custom_app_drop_events` in `create_qt_mime_data` module's docstring like "vertices" and "decorations" are?

Reply:
Done